sandbox/templates/richie/section/_highlighted_items.scss

```diff
@@ -0,0 +1,94 @@
+$richie-section-highlights-padding: 3rem 1rem !default;
+$richie-section-highlights-background-even: $white !default;
+$richie-section-highlights-background-odd: $gray98 !default;
+$richie-section-highlights-title-color: $gray17 !default;
+$richie-section-highlights-title-margin: 0 auto 1rem !default;
+$richie-section-highlights-title-padding: 1rem !default;
+$richie-section-highlights-title-textalign: center !default;
+$richie-section-highlights-item-gutter: 0.625rem !default;
+
+// TODO: unrelated: Better Hero on large_banner template "hero-intro.html"
+.section{
+    $base-section-selector: &;
```

Comment:
Why bother with this? `&` is already a constant

Reply:
Because to be able to make full BEM class name path you need to have the previous parent name, once you enter a new rule like `&--highlights` the value of `&` become `section--highlights` but that's not what we need for such child element like `__title` etc..

Reply:
I guess the plain way I'd write this to be clear would be:
```scss
.section {
  &__title {
    // default styles

    .section--highlights & {}
  }
}
```

I guess in this case it does not really matter because there are no default styles to section titles right?

Reply:
It should do the trick for this context even i'm not very confident to do this everywhere since we will have so to re-open the modifier under each element. This is a personal point of view but commonly i prefer to have one way tree like.

Demonstration: https://gist.github.com/sveetch/f67a7a19678572c1f99d6462619c2e28

Reply:
IMO the point is that the repetition is worth it if it makes the code and its intention trivial to understand.

And (more importantly) its `section__title` styling itself for various circumstances, you don't have to look around modifiers for `section__title` styles.

Wondering what you think about this gist @sampaccoud () ?

Reply:
IMO #pim is much clearer than #poum... title and items are higher levels of details than highlights and dummy, so they should definitely be the parents.

The $ variable name also brings an inconsistence in the syntax style that is difficult to read and understand.